go.mod

```diff
@@ -3,21 +3,21 @@ module sigs.k8s.io/kueue
 go 1.18
 
 require (
-	github.com/go-logr/logr v1.2.2
-	github.com/google/go-cmp v0.5.7
+	github.com/go-logr/logr v1.2.3
+	github.com/google/go-cmp v0.5.8
 	github.com/onsi/ginkgo/v2 v2.1.4
-	github.com/onsi/gomega v1.19.0
+	github.com/onsi/gomega v1.20.0
 	github.com/open-policy-agent/cert-controller v0.3.0
-	github.com/prometheus/client_golang v1.12.1
-	go.uber.org/zap v1.21.0
-	k8s.io/api v0.23.4
-	k8s.io/apimachinery v0.23.4
-	k8s.io/client-go v0.23.4
-	k8s.io/component-base v0.23.3
-	k8s.io/component-helpers v0.23.4
-	k8s.io/klog/v2 v2.40.1
+	github.com/prometheus/client_golang v1.13.0
+	go.uber.org/zap v1.22.0
+	k8s.io/api v0.24.3
+	k8s.io/apimachinery v0.24.3
+	k8s.io/client-go v0.24.3
+	k8s.io/component-base v0.24.3
+	k8s.io/component-helpers v0.24.3
+	k8s.io/klog/v2 v2.60.1
 	k8s.io/utils v0.0.0-20220210201930-3a6ce19ff2f9
-	sigs.k8s.io/controller-runtime v0.11.1
+	sigs.k8s.io/controller-runtime v0.12.3
```

Comment:
was this the only block to be updated manually for dependency update?

Reply:
`require`? yes. The rest was done by `go mod tidy`